modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookManagementToolbarDisplayContext.java

```diff
@@ -166,6 +187,47 @@ public CreationMenu getCreationMenu() {
 		).build();
 	}
 
+	public List<Map<String, Object>> getFrontendTokenDefinitionProviders() {
```

Comment:
Is this method used on other places? If not, we can make it private.

Reply:
It was only used in this file, so I made the change.